script-templates/OpenTelemetry.DotNet.Auto.psm1.template

```diff
@@ -22,6 +22,16 @@ function Get-Current-InstallDir() {
     return [System.Environment]::GetEnvironmentVariable($ServiceLocatorVariable, [System.EnvironmentVariableTarget]::Machine)
 }
 
+function Get-Newest-InstallVersion() { 
+    $query = "https://api.github.com/repos/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest"
+
+    $response = Invoke-WebRequest -Uri $query
```

Comment:
It is good to set `-UseBasicParsing` in all `Invoke-WebRequest` so it works on older versions of Windows Server.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3042/commits/d26a4cf7d7569e883ec73751da100b49826d8b7b